lib/web_ui/lib/src/engine/canvas_pool.dart

```diff
@@ -865,54 +865,54 @@ class ContextStateHandle {
       _debugIsPaintSetUp = false;
     }
 
-    final ui.MaskFilter maskFilter = _lastUsedPaint?.maskFilter;
+    final ui.MaskFilter? maskFilter = _lastUsedPaint?.maskFilter;
     if (maskFilter != null && _renderMaskFilterForWebkit) {
       // On Safari (WebKit) we use a translated shadow to emulate
       // MaskFilter.blur. We use restore to undo the translation and
       // shadow attributes.
-      context.restore();
+      context!.restore();
     }
   }
 
-  void paint(ui.PaintingStyle style) {
+  void paint(ui.PaintingStyle? style) {
     if (style == ui.PaintingStyle.stroke) {
-      context.stroke();
+      context!.stroke();
     } else {
-      context.fill();
+      context!.fill();
     }
   }
 
-  void paintPath(ui.PaintingStyle style, ui.PathFillType pathFillType) {
+  void paintPath(ui.PaintingStyle? style, ui.PathFillType pathFillType) {
     if (style == ui.PaintingStyle.stroke) {
-      context.stroke();
+      context!.stroke();
     } else {
       if (pathFillType == ui.PathFillType.nonZero) {
-        context.fill();
+        context!.fill();
       } else {
-        context.fill('evenodd');
+        context!.fill('evenodd');
       }
     }
   }
 
   void reset() {
-    context.fillStyle = '';
+    context!.fillStyle = '';
```

Comment:
Suggestion: html.CanvasRenderingContext2D ctx = context!; and removing all null checks below.

Reply:
Done.